tests/Unit/UtilizationMetricsApisTest.php

```diff
@@ -0,0 +1,436 @@
+
+/*
+<COPYRIGHT>
+
+    Copyright © 2016-2024, Canyon GBS LLC. All rights reserved.
+
+    Advising App™ is licensed under the Elastic License 2.0. For more details,
+    see https://github.com/canyongbs/advisingapp/blob/main/LICENSE.
+
+    Notice:
+
+    - You may not provide the software to third parties as a hosted or managed
+      service, where the service provides users with access to any substantial set of
+      the features or functionality of the software.
+    - You may not move, change, disable, or circumvent the license key functionality
+      in the software, and you may not remove or obscure any functionality in the
+      software that is protected by the license key.
+    - You may not alter, remove, or obscure any licensing, copyright, or other notices
+      of the licensor in the software. Any use of the licensor’s trademarks is subject
+      to applicable law.
+    - Canyon GBS LLC respects the intellectual property rights of others and expects the
+      same in return. Canyon GBS™ and Advising App™ are registered trademarks of
+      Canyon GBS LLC, and we are committed to enforcing and protecting our trademarks
+      vigorously.
+    - The software solution, including services, infrastructure, and code, is offered as a
+      Software as a Service (SaaS) by Canyon GBS LLC.
+    - Use of this software implies agreement to the license terms and conditions as stated
+      in the Elastic License 2.0.
+
+    For more information or inquiries please visit our website at
+    https://www.canyongbs.com or contact us via email at legal@canyongbs.com.
+
+</COPYRIGHT>
+*/
+
+use App\Models\User;
+
+use function Pest\Laravel\get;
+
+use AdvisingApp\Ai\Models\Prompt;
+use AdvisingApp\Form\Models\Form;
+use AdvisingApp\Task\Models\Task;
+use App\Settings\LicenseSettings;
+use AdvisingApp\Ai\Models\AiThread;
+use AdvisingApp\Alert\Models\Alert;
+use AdvisingApp\Ai\Models\PromptUse;
+use AdvisingApp\Survey\Models\Survey;
+use AdvisingApp\Segment\Models\Segment;
+use App\Http\Middleware\CheckOlympusKey;
+use AdvisingApp\Campaign\Models\Campaign;
+use AdvisingApp\Prospect\Models\Prospect;
+use AdvisingApp\Form\Models\FormSubmission;
+use AdvisingApp\MeetingCenter\Models\Event;
+
+use function Pest\Laravel\withoutMiddleware;
+
+use AdvisingApp\Report\Enums\TrackedEventType;
+use AdvisingApp\Campaign\Models\CampaignAction;
+use AdvisingApp\Survey\Models\SurveySubmission;
+use AdvisingApp\Authorization\Enums\LicenseType;
+use AdvisingApp\Report\Models\TrackedEventCount;
+use AdvisingApp\StudentDataModel\Models\Student;
+use AdvisingApp\ResourceHub\Models\ResourceHubArticle;
+
+beforeEach(function () {
+    withoutMiddleware(CheckOlympusKey::class);
+});
+
+it('checks the API returns users', function () {
+    $randomRecords = random_int(1, 10);
+
+    User::factory()->count($randomRecords)->create();
+
+    $softDeleteUser = User::factory()->create();
+    $softDeleteUser->delete();
+
+    $response = get('/api/utilization-metrics');
```

Comment:
We should be using the route name here, not calling the path directly.

Reply:
Done